drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -15,12 +19,14 @@ type endpointTable map[string]*endpoint
 const overlayEndpointPrefix = "overlay/endpoint"
 
 type endpoint struct {
```

Comment:
Eventually, we should look at converging type endpoint struct and type hnsEnpoint struct in windows pkg so they can be shared by all windows and overlay drivers.  But, I think that is outside the scope of this PR.